public/locales/en/translation.json

```diff
@@ -76,5 +77,21 @@
   "labelPosition": "Label Position",
   "table": {
     "labels": ["Date", "Hour", "Type", "Killed", "Severe injured", "Light injured"]
+  },
+  "login": "LOGIN",
+  "filterPanel": {
+    "all": "all"
+  },
+  "sideBar": "Approximate location:",
+  "UserProfileHeader": {
+    "hallo": "Hallo",
+    "logout": "LOGOUT"
+  },
+  "widgets": {
+    "countByCarType": {
+      "header1": "Accidents count by vehicle type",
```

Comment:
keys should be informative and similar to text (so we can read them in case i18n doesn't work)
for example:
```
"widgets": {
    "countByCarType": {
      "Accidents count by vehicle type": "Accidents count by vehicle type",
      }
      
```

Reply:
I changed the leys to represent the meaning of the written text and updated the branch